bika/lims/exportimport/instruments/facscalibur/calibur/model_e9750.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of SENAITE.CORE
+#
+# Copyright 2018 by it's authors.
+# Some rights reserved. See LICENSE.rst, CONTRIBUTORS.rst.
+
+""" Facs Calibur
+"""
+from bika.lims import bikaMessageFactory as _
+from bika.lims.utils import t
+
+from bhp.lims import PRODUCT_NAME
```

Comment:
This import is making the tests fail: https://travis-ci.org/senaite/senaite.core/builds/503517195?utm_source=github_status&utm_medium=notification
Please, remove it. I think `PRODUCT_NAME` is not used anywhere here.

Reply:
ok i removed it, thanks it was mistakenly included